cl/api/utils.py

```diff
@@ -732,28 +732,43 @@ def invert_user_logs(
     :return The inverted dictionary
     """
     r = get_redis_interface("STATS")
-    pipe = r.pipeline()
 
     dates = make_date_str_list(start, end)
-    for d in dates:
-        pipe.zrange(f"api:v3.user.d:{d}.counts", 0, -1, withscores=True)
-    results = pipe.execute()
+
+    versions = ["v3", "v4"]
+
+    for version in versions:
+        pipe = r.pipeline()
```

Comment:
I don't see how this could work. You establish a new pipe for each version, so this would always throw away the pipe (and it's results) for v3.

Reply:
Ah you're correct, I'm sorry for the silly mistake there.